ci-operator/step-registry/rosa/tests/rosa-tests-ref.yaml

```diff
@@ -35,6 +35,9 @@ ref:
   - name: TEST_TIMEOUT
     default: "4h"
     documentation: Set time for the testing execution.
+  - name: TIMEOUT
+    default: "6h"
+    documentation: Set pod level timeout
```

Comment:
You made an incorrect update. This `timeout` is one of the pod attribute but not the global variable.
 E.g. https://github.com/openshift/release/blob/80c8d40788087fd7a6a7441f934e060d1587d56f/ci-operator/step-registry/rosa/cluster/wait-ready/cluster/rosa-cluster-wait-ready-cluster-ref.yaml#L10